contrib/database/sql/option.go

```diff
@@ -8,16 +8,27 @@ package sql
 import (
 	"math"
 
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 	"gopkg.in/DataDog/dd-trace-go.v1/internal"
 )
 
 type config struct {
-	serviceName    string
-	analyticsRate  float64
-	dsn            string
-	childSpansOnly bool
+	serviceName          string
+	analyticsRate        float64
+	dsn                  string
+	childSpansOnly       bool
+	commentInjectionMode commentInjectionMode
 }
 
+// commentInjectionMode represents the mode of sql comment injection
+type commentInjectionMode int
+
+const (
+	commentInjectionDisabled      commentInjectionMode = 0 // Default value, sql comment injection disabled
```

Comment:
nit: any reason to not use `iota` here? (Not really important, just curious if there's a reasoning)